src/main/resources/static/api.yml

```diff
@@ -1032,6 +1032,26 @@ paths:
           $ref: '#/components/responses/403Response'
         500:
           $ref: '#/components/responses/500Response'
+  /reference-data/local-authorities:
```

Comment:
I think maybe we ought to be consistent with the term "local authority **areas**"?